app/containers/Root.js

```diff
@@ -3,14 +3,24 @@ import { Provider } from 'react-redux';
 import { Router } from 'react-router';
 import routes from '../routes';
 
+// Partials
+import Navbar from '../components/partials/Navbar';
+import Header from '../components/partials/Header';
+import Footer from '../components/partials/Footer';
+
 export default class Root extends Component {
   render() {
     const { store, history } = this.props;
     return (
         <Provider store={store}>
-        <div>
-        <Router history={history} routes={routes} />
-        </div>
+          <div className="container">
```

Comment:
Definition for class `container` doesn't seem to be anywhere in the css. Either add the css rule, if required or else remove this class altogether.